src/NLight.Tests.Benchmarks/IO/Text/DelimitedRecordReaderBenchmarks.cs

```diff
@@ -124,9 +126,11 @@ public static void ReadAll_CsvHelper(DelimitedRecordReaderBenchmarkArguments arg
 				{
 					while (reader.Read())
 					{
-						var record = reader.Context.Record;
-						for (int i = 0; i < record.Length; i++)
-							s = record[i];
+						IDictionary<string,Object> record = (IDictionary<string, Object>)reader.GetRecord<dynamic>();
```

Comment:
Isn't there a more elegant (and faster) way to read values from the record?

Reply:
Sorry, I´m not used to the CsvHelper library. I just tried to make the code working. Should I downgrade the CsvHelper nuget package to the old version, so that the original code is working again?

Reply:
Looking at CsvHelper Reader class source, I think new code should be (did not test it)

```
while (reader.Read())
{
    for (int i = 0; i < reader.ColumnCount; i++)
        s = reader[i];
}
```

Reply:
Yes, it´s working with this code.